_posts/2016/2016-05-31-gradle-jenkins-plugin.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: post
+title: "How to empower your Jenkins with Gradle"
+date: "2016-05-31 10:59:11"
+categories: tech-stories
+authors: ["Renato T."]
+---
+
+Since the [last blog]( https://developer.epages.com/blog/2016/01/28/jenkins-job-dsl-plugin.html) post we did some updates in our Jenkins environments and today we are going to talk a bit about it.
+We are using a [Gradle Jenkins plugin](https://github.com/ghale/gradle-jenkins-plugin/wiki) to control Jenkins installation.
+
+We already mention about the dsl, in the [Job DSL Plugin](https://wiki.jenkins-ci.org/display/JENKINS/Job+DSL+Plugin)
+
+So you may ask why we changed it?
+Well, the other plugin you need to create a seed job, and at ePages, we have quite complex structure with a few shell scripts.
+Some of those shell scripts are shared on the company level, so before changing them, we want to make sure they work properly. (testing fase!)
+
+Another advantage of this gradle plugin is that is simple to deploy. For instance I didn't want to break our Team Jenkins installation, so I run a instance of Jenkins in a docker container and deploy my playground scripts to it.
+
+## How to setup the project
+In your main Project you should use the [gradle-jenkins-plugin](https://github.com/ghale/gradle-jenkins-plugin/wiki).
+
+I created a sample snippet for writing this blog post:
```

Comment:
I created a sample snippet for writing this blog post:
>
Look at the following simple snippet: